packages/developer-portal/src/components/pages/settings/settings-billing-tab/accounts-information-form/helper-text.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import { cx } from 'linaria'
+import { isBlue } from './__styles__/helper-text'
+
+export type HelperTextProps = {
+  text: string
+}
+
+const HelperText: React.FC<HelperTextProps> = ({ text }) => {
```

Comment:
I can we not just use https://elements.reapit.cloud/?path=/story/helper--info here? I think this was Holly's intention

Reply:
Updated